HucaresWF/src/Hucares.Server.Client/MissingPlateClient.cs

```diff
@@ -0,0 +1,159 @@
+
+            var plateNumberData = new
+            {
+                plateNumber = plateNumber,
+                searchStartDatetime = searchStartDatetime
+            };
+
+            var jsonContent = JsonConvert.SerializeObject(plateNumberData, new JsonSerializerSettings
+            {
+                NullValueHandling = NullValueHandling.Ignore
+            });
+            
+            var request = new HttpRequestMessage(HttpMethod.Post, fullUri)
+            {
+                Content = new StringContent(jsonContent, Encoding.UTF8, "application/json")
+            };
+
+            return await httpHelper.MakeRequest<MissingLicensePlate>(request);
+        }
+
+        public async Task<IEnumerable<MissingLicensePlate>> GetAllPlateRecords()
+        {
+            var uri = $"api/mlp/all";
+            var fullUri = new Uri(HostUri, uri);
+            var request = new HttpRequestMessage(HttpMethod.Get, fullUri);
+            
+            return await httpHelper.MakeRequest<IEnumerable<MissingLicensePlate>>(request);
+        }
+
+        public async Task<IEnumerable<MissingLicensePlate>> GetPlateRecordByPlateNumber(string plateNumber)
+        {
+            var uri = $"api/mlp/plate/{plateNumber}";
+            var fullUri = new Uri(HostUri, uri);
+            
+            var jsonContent = JsonConvert.SerializeObject(plateNumber, new JsonSerializerSettings
+            {
+                NullValueHandling = NullValueHandling.Ignore
+            });
+            
+            var request = new HttpRequestMessage(HttpMethod.Post, fullUri)
+            {
+                Content = new StringContent(jsonContent, Encoding.UTF8, "application/json")
+            };
+            
+            return await httpHelper.MakeRequest<IEnumerable<MissingLicensePlate>>(request);
+        }
+
+        public async Task<MissingLicensePlate> UpdatePlateRecord(int plateId, string plateNumber, DateTime searchStartDatetime)
+        {
+            var uri = $"api/mlp/update/{plateId}";
+            var fullUri = new Uri(HostUri, uri);
+
+            var plateNumberData = new
+            {
+                plateId = plateId,
+                plateNumber = plateNumber,
+                searchStartDatetime = searchStartDatetime
+            };
+            
+            var jsonContent = JsonConvert.SerializeObject(plateNumberData, new JsonSerializerSettings
+            {
+                NullValueHandling = NullValueHandling.Ignore
+            });
+            
+            var request = new HttpRequestMessage(HttpMethod.Post, fullUri)
+            {
+                Content = new StringContent(jsonContent, Encoding.UTF8, "application/json")
+            };
+
+            return await httpHelper.MakeRequest<MissingLicensePlate>(request);
+        }
+
+        public async Task<MissingLicensePlate> MarkFoundPlate(int plateId, DateTime requestDateTime, bool isFound)
+        {
+            var uri = $"api/mlp/found/{plateId}/{isFound}";
```

Comment:
This is a mistake in backend. isFound is already in the body so it is not needed in uri